packages/components/app-icon/src/app-icon.js

```diff
@@ -8,8 +8,11 @@ export class TSIcon extends TSElement {
 
 	static get properties() {
 		return {
+			/** Size of the app icon, available variants: ''(default), 'large' */
```

Comment:
Do you think we can use [this](https://github.com/jsdoc/jsdoc/issues/629#issue-31314808) approach to document limited options for params? E.g.:
```js
/**
* Size of the app icon.
* @type {('default'|'large')}
*/
```
My intention with this proposal is to have a standard way to document limited options, because I think this would help with generating `d.ts` files later.

Reply:
I agree, it should be like that, I had something similar in mind, but deliberately didn't included it in this PR. I think we should create the guideline for consistency and include when and how these enums should be used. If you think it's better to do in this PR I will do it.

Reply:
Ok, sounds good to do it in a separate PR. I just want to be sure that we are on the same page :)